sessions/js-structure/quiz-app/components/card-list/card-list.css

```diff
@@ -0,0 +1,9 @@
+.card-list {
+  padding: 10rem 2rem 4rem 2rem;
+  max-width: 45rem;
```

Comment:
Should we be using rem here? I don't really see any benefit in using a relative unit for a max-width here.